posts/2023-12-12-23.0.0.12.adoc

```diff
@@ -0,0 +1,533 @@
+
+
+The following example expands on the previous example to define histogram buckets for the `alpha.timer` timer metric by using the `mp.metrics.distribution.timer.buckets` property:
+
+
+[source,xml]
+----
+mp.metrics.distribution.timer.buckets=alpha.timer=100ms,200ms,1s
+----
+
+This configuration tells the metrics runtime to track and output the count of durations that fall within 0-100ms, 0-200ms, and 0-1 seconds. These values are ranges because the histogram buckets work cumulatively.
+
+
+The corresponding Prometheus output for the `alpha.timer` metric at the `/metrics` REST endpoint is:
+
+[source]
+----
+# HELP alpha_timer_seconds_max
+# TYPE alpha_timer_seconds_max gauge
+alpha_timer_seconds_max{scope="application",} 5.633
+# HELP alpha_timer_seconds
+# TYPE alpha_timer_seconds histogram <1>
+alpha_timer_seconds{scope="application",quantile="0.5",} 0.67108864
+alpha_timer_seconds{scope="application",quantile="0.7",} 5.603590144
+alpha_timer_seconds{scope="application",quantile="0.75",} 5.603590144
+alpha_timer_seconds{scope="application",quantile="0.8",} 5.603590144
+alpha_timer_seconds_bucket{scope="application",le="0.1",} 0.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="0.2",} 0.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="1.0",} 1.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="+Inf",} 2.0  <2><3>
+alpha_timer_seconds_count{scope="application",} 2.0
+alpha_timer_seconds_sum{scope="application",} 6.333
+----
+
+<1> The Prometheus metric type is `histogram`. Both the quantiles or percentiles and buckets are represented under this type.
+<2> The `le` tag represents _less than_ and is for the defined buckets, which are converted to seconds.
+<3> Prometheus requires a `+Inf` bucket, which counts all hits.
+
+Additionally, the `@RegistryScope` annotation is now a CDI qualifier.
+
+For more information about MicroProfile Metrics, see:
+
+* link:https://github.com/eclipse/microprofile-metrics[MicroProfile Metrics repo]
+* link:{url-prefix}/docs/latest/introduction-monitoring-metrics.html[Monitoring with metrics documentation]
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26945
+// Contact/Reviewer: yasmin-aumeeruddy
+// // // // // // // //
+[#mpt11]
+=== Expanded Java EE and Jakarta EE support in MicroProfile Telemetry 1.1
+
+MicroProfile Telemetry 1.1 provides you with the latest Open Telemetry technology as the feature now consumes OpenTelemetry-1.29.0, which is updated from 1.19.0.
+
+The feature is compatible with the following programming model combinations:
+
+* Java EE 7 paired with MicroProfile 1.4
+* Java EE 8 paired with MicroProfile 4.1
+* Jakarta EE 9 paired with MicroProfile 5.0
+* Jakarta EE 10 paired with MicroProfile 6.1.
```

Comment:
I'm not sure why but these bullet points aren't formatted properly in the draft site.

Reply:
Hi @yasmin-aumeeruddy - this list was fixed in [apply suggestions from review](https://github.com/OpenLiberty/blogs/pull/3470/commits/c14691fdc2a11d3aa2a1aa7e7bbc08d9e0d76dc6) but the changes haven't been committed back to the draft site yet. I will be sure to verify them once the staging PR is merged